src/StepDefinition/DrupalFormJsEntityBrowserTrait.php

```diff
@@ -78,10 +80,10 @@ public function iClickOnItemWithNameInEntityInBrowser($name, $entity_browser) {
   }
 
   /**
-   * @Given Item with label :label in entity browser :entity_browser should have the class :class_name
+   * @Given Item with label :label in entity browser :entity_browser in field with class :title_field_class should have the class :class_name
    */
-  public function itemInEntityBrowserShouldHaveClass($label, $enity_browser, $class_name) {
-    $item_selector = $this->getEntityBrowserItemSelector($label);
+  public function itemInEntityBrowserShouldHaveClass($label, $enity_browser, $title_field_class, $class_name) {
```

Comment:
This is a back-wards compatibility issue.

Just add $title_field_class = 'old-value' as last parameter. Behat will pass the parameter correctly based upon naming.
Then have both, the old + the new @Given gherkin syntax supported by adding both lines. Examples for that strategy working is in the mink element extension trait.